### ISSUE TITLE
In english documents,there are some mistakes of describtion of the parameter.

### ISSUE THREAD

**Issue by jiacy-jcy** [taosdata/TDengine#7954]
![图片](https://user-images.githubusercontent.com/57592088/133568157-a517591f-76f0-45cc-9331-2db649708571.png)

![图片](https://user-images.githubusercontent.com/57592088/133568304-00199a07-098b-417c-a65d-3128d42e0ded.png)
In the first picture,the parameter described under "  ".
But in the second picture,the parameter describtion is different from the first.
In fact, the first one is true.We need somebody to help us to modify the english documents.
Here is the document link:  
https://github.com/taosdata/TDengine/blob/develop/documentation20/en/03.architecture/docs.md

**Comment by RedemptionC** [comment]
hello,maybe I can try this.

**Comment by RedemptionC** [comment]
do you mean all parameters decorated with `  , should be decorated with " instead?
because there are many similary cases:
![image](https://user-images.githubusercontent.com/17563754/134459121-3d3021a7-68d1-4667-833b-cc1b8ab75d96.png)
![image](https://user-images.githubusercontent.com/17563754/134459148-34ce5ba8-b7f0-4add-9048-387252ced11c.png)

Or do we only need to modify the posted case?
![image](https://user-images.githubusercontent.com/17563754/134459330-22a77d74-78d7-4a75-86d3-64917f8822ba.png)

**Comment by jiacy-jcy** [comment]
> 
> 
> do you mean all parameters decorated with ` , should be decorated with " instead?
> because there are many similary cases:
> ![image](https://user-images.githubusercontent.com/17563754/134459121-3d3021a7-68d1-4667-833b-cc1b8ab75d96.png)
> ![image](https://user-images.githubusercontent.com/17563754/134459148-34ce5ba8-b7f0-4add-9048-387252ced11c.png)
> 
> Or do we only need to modify the posted case?
> ![image](https://user-images.githubusercontent.com/17563754/134459330-22a77d74-78d7-4a75-86d3-64917f8822ba.png)

you should modify the all parameters decorated with " " around the whole document,which described in 'configuration parameter'.

**Comment by Aries-Lee1991** [comment]
> Please assign to me. By the way, do I need to modify parameters other than 'configuration parameter'？

Assigned it to you.

**Comment by Aries-Lee1991** [comment]
> Please assign to me. By the way, do I need to modify parameters other than 'configuration parameter'？

@jiacy-jcy 看一下这个问题哈。

**Comment by jiacy-jcy** [comment]
> 
> 
> Please assign to me. By the way, do I need to modify parameters other than 'configuration parameter'？

You are right. The other configuration parameters should be modified the same as the picture.
